docs/docs/fallback-handoff.mdx

```diff
@@ -283,6 +285,44 @@ rules:
   - active_loop: action_two_stage_fallback
 ```
 
+### 4. Defining an ultimate fallback action (optional)
+
+To define the bot's response when the user denies the rephrased intent, define the response `utter_default`:
+
+```yaml-rasa title="domain.yml"
+responses:
+  utter_default:
+  - text: I'm sorry, I can't help you.
+```
+
+Or, you can customize `action_default_fallback` for more complex behavior by writing a [Custom Action](./actions.mdx#custom-actions). 
+For example, the following snippet is an implementation of a custom action which does the same as `action_default_fallback` but 
+dispatches a different template `utter_fallback_template`:
+
+```python title="actions.py"
+from typing import Any, Dict, List, Text
+
+from rasa_sdk import Action, Tracker
+from rasa_sdk.events import UserUtteranceReverted
+from rasa_sdk.executor import CollectingDispatcher
+
+class ActionDefaultFallback(Action):
+    def name(self) -> Text:
+        return "action_default_fallback"
+
+    def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_message(template="my_custom_fallback_template")
+
+        # Revert user message which led to fallback.
+        return [UserUtteranceReverted()]
```

Comment:
As I rediscovered today, keeping this `UserUtteranceReverted()` in the return events is crucial to not ending up with repeated fallback actions leading to the bot not responding - I forget what it's called, but when it endlessly predicts the same action.

If you can verify this behaviour, we should document it clearly so people don't overwrite the action and get unexpected behaviour.

Reply:
Sorry should have clarified: If you **do not** return `UserUtteranceReverted()` ,  you will see this "catastrophic backtracking"

Reply:
Sorry should have clarified: If you **do not** return `UserUtteranceReverted()` ,  you will see this "catastrophic backtracking"

Reply:
Maybe it was something specific to the way I tried it 🤷‍♀️

Reply:
yes, removing the TED Policy was the root cause.

Reply:
Right, we do still need overwriting behaviour to work in the absence of TEDPolicy though, so root-root cause is something more.

Reply:
I think with your explanation below we can resolve this at any rate